.kokoro/publish.sh

```diff
@@ -28,4 +28,24 @@ NPM_TOKEN=$(cat $KOKORO_KEYSTORE_DIR/73713_google-cloud-npm-token-1)
 echo "//wombat-dressing-room.appspot.com/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
 npm install
-npm publish --access=public --registry=https://wombat-dressing-room.appspot.com
+
+# First, pack this project as a tarball
+
+npm pack .
+
+# Second, publish the tarball that was just created
+
+# npm provides no way to specify, observe, or predict the name of the tarball
+# file it generates.  We have to look in the current directory for the freshest
+# .tgz file.
+TARBALL=$(ls -1 -t *.tgz | head -1)
+npm publish --access=public --registry=https://wombat-dressing-room.appspot.com "$TARBALL"
```

Comment:
`npm pack` actually output the file name to stdout, so this should be feasible with something like: 
```suggestion

# First, pack this project as a tarball

TARBALL_BASENAME=$(npm pack) .

# Second, publish the tarball that was just created

TARBALL="$(pwd)/$TARBALL_BASENAME"
npm publish --access=public --registry=https://wombat-dressing-room.appspot.com "$TARBALL"
```